src/js/base/CloseButton.tsx

```diff
@@ -7,3 +7,6 @@ const StyledButton = styled(Icon)`
 `;
 
 export const CloseButton = ({ onClick }) => <StyledButton name="times" onClick={onClick} />;
+
+
+
```

Comment:
Reduce to one line.

Reply:
fixed. Sorry I didn't realize that prettier was not configured for typescript files